README.md

```diff
@@ -283,10 +283,36 @@ And finally, you are ready to go! Or at least you should be...
 
 ### Junior Contributors
 
+**Gemma Sagastegui** | [<gemma.sagastegui@alum.esci.upf.edu>][sagggastegui] | CoffeeL. 
```

Comment:
In the first square bracket you should put your name or  github username. The second one, must have the same name as the link you provide (you already did it correctly)